src/inspect_ai/tool/_tools/_web_browser/_web_browser.py

```diff
@@ -57,7 +74,9 @@ def web_browser_go() -> Tool:
     async def execute(url: str) -> str:
         """Navigate the web browser to a URL.
 
-        Once you have navigated to a page, you will be presented with a web accessibilty tree of the elements on the page. Each element has an ID, which is displayed in brackets at the beginning of its line. For example:
+        Once you have navigated to a page, you will be presented with HTML or a web accessibility tree of the elements on the page.
```

Comment:
Here we would need to do dynamic prompting based on the output format chosen (note that the API above also implies that DOM and Pixels are supported -- not sure if they are I'd lean against for now -- but we'd need to prompt for them as well if they were in play). 

Note that dynamic prompting for interactive vs not interactive is done here: https://github.com/UKGovernmentBEIS/inspect_ai/blob/cd99abb09d4099030a667fd555cc0629624ef21b/src/inspect_ai/tool/_tools/_web_browser/_web_browser.py#L43. We'd need to do something similar for the output format. At some point it will become easier to just use ToolDefs for this: https://inspect.ai-safety-institute.org.uk/tools.html#sec-dynamic-tools